packages/cli/README.md

```diff
@@ -31,6 +31,12 @@ As this prints out to `process.stdout` it is possible to pipe this output to ano
 ogma production.log > production.hydrated.log
 ```
 
+You can also read from a stream. For example, you can redirect the output of `tail` to read incoming lines:
+
+```sh
+tail -F production.log | ogma hydrate
+```
```

Comment:
Can we make sure to mention that `tail` is a `unix` command (or does it exist in Windows too?)

Reply:
I have added that now. PowerShell apparently [has `tail`-like functionality](https://stackoverflow.com/a/188126), but I don't have means of testing that it actually works.